src/Tecan/Rack/BaseRack.php

```diff
@@ -0,0 +1,85 @@
+<?php declare(strict_types=1);
+
+namespace MLL\Utils\Tecan\Rack;
+
+use Illuminate\Support\Collection;
+
+abstract class BaseRack implements Rack
+{
+    public const EMPTY_POSITION = null;
+
+    /** @var Collection<int, mixed> */
+    public Collection $positions;
+
+    public function __construct()
+    {
+        $this->positions = Collection::times($this->positionCount(), fn () => self::EMPTY_POSITION)
+            ->mapWithKeys(fn ($content, int $position): array => [$position + 1 => $content]);
+    }
+
+    public function id(): ?string
+    {
+        return null;
+    }
+
+    public function toString(): string
+    {
+        return implode(';', [
+            $this->name(),
+            $this->id(),
+            $this->type(),
+        ]);
+    }
+
+    /** @param string|int|object|null $content */
```

Comment:
Wieder inkonsistent mit der Angabe des Property:

```
    /** @var Collection<int, mixed> */
    public Collection $positions;
```

Warum nicht einfach nur `mixed`?

Reply:
sieht Kommentar oben. `/** @param mixed $content */` wird vom php-fixer gelöscht.. ok, jetzt sehe ich es mit dem lesbaren Kommentar in dem PHP doc nicht vom Fixer gelöscht wird.